webapp/channels/src/components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -239,66 +375,77 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
         }) as UserProfileValue[];
     };
 
-    renderOption = (option: UserProfileValue, isSelected: boolean, onAdd: (user: UserProfileValue) => void, onMouseMove: (user: UserProfileValue) => void) => {
+    renderOption = (option: UserProfileValue | GroupValue, isSelected: boolean, onAdd: (option: UserProfileValue | GroupValue) => void, onMouseMove: (option: UserProfileValue | GroupValue) => void) => {
```

Comment:
Any time you write `render[Somthing]` you should probably be writing a separate component. Pull this out into a functional component?

Reply:
Yeah sure!